.github/actions/deploy/action.yml

```diff
@@ -32,5 +32,5 @@ runs:
       working-directory: voluncheer
       run: |
         eb init ${{ inputs.app-name }} --region ${{ inputs.region }} --platform ${{ inputs.platform }}
-        eb deploy ${{ inputs.app-name }}-${{ github.ref_name }} --label ${{ inputs.app-name}}_${{ github.ref_name }}_${{ steps.timestamp.outputs.TIMESTAMP }}
+        eb deploy ${{ inputs.app-name }}-${{ github.ref_name }}-v2 --label ${{ inputs.app-name}}_${{ github.ref_name }}_${{ steps.timestamp.outputs.TIMESTAMP }}
```

Comment:
So we are deploying to main-v2 and develop-v2?

Did you give the right permission to the two deployment roles?

Reply:
I did last time for dev. I am doing the same for main.

Reply:
Did you give the right permission to the two deployment roles?

Reply:
I think I did.